### ISSUE TITLE
Installation failed

### ISSUE THREAD

**Issue by alex9434** [tpartner/LimeSurvey-Net-Promoter-Score#6]
I am on LimeSurvey 3.x. I have installed the archive at /pathToLimeSurvey/upload/themes/question/ as per your instruction. I have created the directory question because it was not present. Unfortunatley, when I select the "List (radio)" I have not NPS survey under "Question Theme". What am I doing wrong?

**Comment by tpartner** [comment]
What version of 3.x? You should not have to create any directories.

Did you place the **_Net-Promoter-Score_** folder in _/pathToLimeSurvey/upload/themes/question/_ as indicated in the instructions?

**Comment by alex9434** [comment]
Yes. I have tried both variants. limesurvey/upload/themes/question/ (as per instruction) and limesurvey/upload/themes/questions/ (plural) as in the actual file. Both directories did not exist and had to be created. There was no question(s) subdirectory under themes. The full path then reads: limesurvey/upload/themes/question/answer/listradio. This was on version 3.23 CE.

I then installed to the "brand-new" version 4.3.33 CE hoping that the theme editor would solve my issue. I followed the instractions downloaded the repository, changed the directory and created a new zip file:
```
cd ~/Downloads/LimeSurvey-Net-Promoter-Score-master/Net-Promoter-Score
zip -9 -r ~/Downloads/nps.zip *
```
I tried to upload this file with the 4.x theme editor but I get the following error message:
`×The ZIP file contains wrong paths`
The theme then shows up under installed themes but does not work

**Comment by tpartner** [comment]
The full path should be:

`/pathToLimeSurvey/upload/themes/question/Net-Promoter-Score/survey/questions/answer/listradio`

It must be the **Net-Promoter-Score** directory, not one of the parent folders that are introduced by the GitHub download process.


![capture_5](https://user-images.githubusercontent.com/1437729/103896008-4a45dc00-50bf-11eb-8fe8-bc3138bcbce1.png)

**Comment by alex9434** [comment]
Hi @tpartner, Thank you very much! That was it. I was not clear that there is question and questions in the directory. That solved my problem on version Version 3.25.6+201229. Apologies that I mis-understood the installation instructions. Maybe someone else makes the same mistake and can profit from your expalation
